### PR TITLE
docs: Update tracked entities ownership [DHIS2-17236]

### PR DESCRIPTION
**Pull request by muilpp**
Update the ACL documentation with the last changes we did on [DHIS2-17236](https://dhis2.atlassian.net/browse/DHIS2-17236)

[DHIS2-17236]: https://dhis2.atlassian.net/browse/DHIS2-17236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by muilpp** [comment]
Since this affects TEs only, I wonder if it would make more sense to include it in the [Tracked Entities section](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html#tracked-entities-get-apitrackertrackedentities) instead. The drawback is that this would result in scattered ACL documentation.

File: src/developer/web-api/tracker.md [hunk 1]
```diff
@@ -2924,6 +2924,12 @@ Tracker objects, the requested organisation unit must always be within the user'
 user cannot request objects outside its search scope unless it's using the organisation unit mode
 ALL and has enough privileges to use that mode.
 
+When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:
+- The tracked entity is enrolled in at least one program the user has data access to, and the user has access to the owner organisation unit.
+- The tracked entity is not enrolled in at least one program the user has data access to, and the user has access to the tracked entity registering organisation unit.
```
Comment (teleivo) [inline src/developer/web-api/tracker.md]: That sounds odd 🤔 

`The tracked entity is not enrolled in at least one program the user has data access to`

Can you phrase this differently? I don't I get it 😅

File: src/developer/web-api/tracker.md [hunk 2]
```diff
@@ -2924,6 +2924,12 @@ Tracker objects, the requested organisation unit must always be within the user'
...
+
+In both scenarios, the validation process will consider the program access level, as detailed at the beginning of this section, and the sharing settings, as explained [here](#webapi_tracker_metadata_sharing).
```
Comment (teleivo) [inline src/developer/web-api/tracker.md]: Can we merge this with 

> When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:

to simplify things i.e. no reference back to the paragraph above.

How about

When requesting tracked entities without specifying a program, the response will include only tracked entities that satisfy [metadata sharing settings](#webapi_tracker_metadata_sharing) and one of the following criteria:

**Review by teleivo — COMMENTED** [review]
I would keep the change in the ACL section. If you want to restructure the docs with regards to ACL we should rethink more than just this ACL addition.

File: src/developer/web-api/tracker.md (hunk 1 above)
Comment (muilpp) [reply src/developer/web-api/tracker.md]: it should be a bit more clear now

File: src/developer/web-api/tracker.md [hunk 3]
```diff
@@ -2924,11 +2924,9 @@ Tracker objects, the requested organisation unit must always be within the user'
...
 ALL and has enough privileges to use that mode.
 
-When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:
+When requesting tracked entities without specifying a program, the response will include only tracked entities that satisfy [metadata sharing settings](#webapi_tracker_metadata_sharing) and one of the following criteria:
 - The tracked entity is enrolled in at least one program the user has data access to, and the user has access to the owner organisation unit.
-- The tracked entity is not enrolled in at least one program the user has data access to, and the user has access to the tracked entity registering organisation unit.
-
-In both scenarios, the validation process will consider the program access level, as detailed at the beginning of this section, and the sharing settings, as explained [here](#webapi_tracker_metadata_sharing).
+- The tracked entity is not enrolled in any program the user has data access to, and the user has access to the tracked entity registering organisation unit.
```
Comment (teleivo) [inline src/developer/web-api/tracker.md]:
```suggestion
- The tracked entity is not enrolled in any program the user has data access to, but the user has access to the tracked entity registering organisation unit.
```

**Review by teleivo — APPROVED** [review]

**Pull request merged**